### ISSUE TITLE
Autocomplete stops at last directory when using `zip`

### ISSUE THREAD

**Issue by Joxtacy** [warpdotdev/Warp#2039]
### Discord username (optional)

Joxtacy#4160

### Describe the bug

When I use the `zip` CLI and want to add several individual files to an archive, the autocomplete stops once it reaches the last directory in the list and will not start autocompleting for files.

This seems to only be an issue with the first argument of files passed to the `zip` CLI.

### To Reproduce

1. Start typing `zip archive.zip`
2. Use suggestions to find the file you want to add.
3. Once you reach the last directory, files will not show up as suggestions.

**Example:**

Have a directory structure like the following.

```
.
└── images
    └── some_file.txt

1 directory, 1 file
```

Warp will suggest `images`, but once that is done, it will not autocomplete for `some_file.txt`.

**Note:** This seems to only happen for the first file argument. Add any more and they will get suggestions as expected.

### Expected behaviour

I would expect Warp to also give suggestions on the files within the directory.

### Screenshots

![Screenshot 2022-11-02 at 17 05 36](https://user-images.githubusercontent.com/10127673/199433589-8e7a14f9-307f-41f9-83cb-c5fd87a6dc44.png)
Example of it not suggesting `some_file.txt`.

![Screenshot 2022-11-02 at 17 07 59](https://user-images.githubusercontent.com/10127673/199434084-77653284-9f10-42df-b288-dec33bdf0659.png)
Example of suggestion working for the second and all other arguments.

### Operating System

MacOS

### OS Version

12.6

### Shell Version

zsh 5.8.1 (x86_64-apple-darwin21.0)

### Warp Version

v0.2022.10.25.08.06.stable_03

### Does this block you from using Warp daily?

No

**Comment by elviskahoro** [comment]
@Joxtacy Is this still happening? If so can you add a video? I'm having trouble reproducing it.

**Comment by Joxtacy** [comment]
@elviskahoro Yes, this is still happening.

Here is a link to the video showing the autocompletion not happening for the second argument to zip, which should be able to be either a directory or a file.

https://imgur.com/a/IRYBb9x

**Comment by Joxtacy** [comment]
Yes, this seems to still be an issue on the latest version of warp (v0.2026.04.08.08.36.stable_05). 

## Steps to reproduce
1. `mkdir /tmp/test-dir`
2. `touch /tmp/test-dir/file.txt`
3. Start typing `zip archive.zip /tmp/test-dir` and try to autocomplete to get the file in the `test-dir` directory. Autocomplete will not recognize the file in there.

## Video trying to illustrate autocomplete not completing the file in `/tmp/test-dir`

https://github.com/user-attachments/assets/394d55d0-6793-4275-93a1-ed7464de8912
